package.json

```diff
@@ -5,9 +5,10 @@
 	],
 	"version": "0.0.1",
 	"private": true,
+	"packageManager": "npm@8.5.1",
 	"scripts": {
 		"cli:create": "npm run dev --workspace=create-ikit-app",
-		"build": "turbo run build",
+		"build": "NODE_OPTIONS=--openssl-legacy-provider turbo run build",
```

Comment:
Should use cross-env or similar (for the 5 windows users)

Reply:
Oh snap we don't need this anymore with the latest release, removing (was temp code anyways)